src/Avalonia.Animation/AnimationInstance`1.cs

```diff
@@ -45,8 +46,9 @@ public AnimationInstance(Animation animation, Animatable control, Animator<T> an
             _onCompleteAction = OnComplete;
             _interpolator = Interpolator;
             _baseClock = baseClock;
-            _neutralValue = (T)_targetControl.GetValue(_animator.Property);
+            control.PropertyChanged += ControlPropertyChanged;
```

Comment:
I think this will keep this instance alive since you are not unsubscribing.

Reply:
Ah damn, I thought I added that but looks like I forgot. Will fix.